<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-PH5L4SJ" ignored="0" name="Test" tests="5" failures="1" timestamp="2022-04-01T23:21:46 IST" time="0.094" errors="0">
  <testcase name="username" time="0.015" classname="org.TestNG.features.I_Annotation_Transformer_concept$Hard_Assertion"/>
  <testcase name="password" time="0.0" classname="org.TestNG.features.I_Annotation_Transformer_concept$Hard_Assertion">
    <skipped/>
  </testcase> <!-- password -->
  <testcase name="password" time="0.0" classname="org.TestNG.features.I_Annotation_Transformer_concept$Hard_Assertion">
    <skipped/>
  </testcase> <!-- password -->
  <testcase name="password" time="0.0" classname="org.TestNG.features.I_Annotation_Transformer_concept$Hard_Assertion">
    <skipped/>
  </testcase> <!-- password -->
  <testcase name="password" time="0.0" classname="org.TestNG.features.I_Annotation_Transformer_concept$Hard_Assertion">
    <failure type="java.lang.AssertionError" message="expected [1234] but found [2345]">
      <![CDATA[java.lang.AssertionError: expected [1234] but found [2345]
at org.TestNG.features.I_Annotation_Transformer_concept$Hard_Assertion.password(I_Annotation_Transformer_concept.java:20)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- password -->
</testsuite> <!-- Test -->
